inc/class-contributors.php

```diff
@@ -153,8 +155,19 @@ public function getArray( $post_id, $contributor_type ) {
 		if ( is_array( $meta ) ) {
 			foreach ( $meta as $slug ) {
 				$name = $this->personalName( $slug );
-				if ( $name ) {
-					$contributors[] = $name;
+				if ( ! $include_term_meta ) {
+					if ( $name ) {
+						$contributors[] = $name;
+					}
```

Comment:
Is this condition necessary?

Reply:
yes, it is. I improved it a little bit in the latest version. But $name is a result of `personalName` method and it could be `''` which means there are not any contributors with that slug. In normal conditions it should not happen, but there is an issue where we left term metadata information after delete a contributor. I addressed that issue here: https://github.com/pressbooks/pressbooks/issues/2385
In summary, if we found the metadata, but we aren't able to see the name, we don't want to list the contributors because it was deleted.

Reply:
Thanks for clarifying it